app/controllers/pregnancies_controller.rb

```diff
@@ -3,6 +3,8 @@ class PregnanciesController < ApplicationController
   rescue_from Mongoid::Errors::DocumentNotFound, with: -> { redirect_to root_path }
 
   def edit
+    @notes = @pregnancy.notes
```

Comment:
In the spirit of query optimizing, going to just draw this from the `pregnancy` obj we're already retrieving and just pass pregnancy into the local.